<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckItemPage</class>
 <widget class="QWidget" name="CheckItemPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_19" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QScrollBar
{
	background:#FFFFFF;
	height:20px;
	padding-left:20px;
	padding-right:20px;
}

QScrollBar::handle
{
	background: #FC8217;
}
QScrollBar::sub-line
{
	background:#FFFFFF;
	border-image: url(:/icons/images/icon_arrow_left.png);
}
QScrollBar::add-line
{
	background:#FFFFFF;
	border-image: url(:/icons/images/icon_arrow_right.png);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_28">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">
background:#122033;




</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1018</width>
           <height>148</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_27">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_tag"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_20" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_20{
background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #F1F6FF , stop:1 #E1EEFF);
}


QPushButton{
border:none;
background:transparent;
}

QPushButton:hover{
background:#46cdcf;
}

QPushButton:pressed{
background:#3d84a8;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>25</number>
      </property>
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/icons/images/add.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/icons/images/delete.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_print">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/icons/images/print.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_pdf">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/icons/images/pdf.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_commit">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/icons/images/commit.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_24" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_31">
        <item>
         <widget class="QWidget" name="widget_21" native="true">
          <property name="styleSheet">
           <string notr="true">QWidget
{
background:transparent;
border:none;
}
QLabel{
color:#1a1a1a;
font-size:16px;
line-height:24px;
}

QLineEdit{
color:#1a1a1a;
font-size:16px;
line-height:24px;
padding-left:8px;
border:none;
border-bottom:1px solid #1a1a1a;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_29">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_23">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string> 科室：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_s_department">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>48</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_22" native="true">
          <property name="styleSheet">
           <string notr="true">QWidget
{
background:transparent;
border:none;
}
QLabel{
color:#1a1a1a;
font-size:16px;
line-height:24px;
}

QLineEdit{
color:#1a1a1a;
font-size:16px;
line-height:24px;
padding-left:8px;
border:none;
border-bottom:1px solid #1a1a1a;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_32">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string> 部位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_s_check_part">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>48</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_23" native="true">
          <property name="styleSheet">
           <string notr="true">QWidget
{
background:transparent;
border:none;
}
QLabel{
color:#1a1a1a;
font-size:16px;
line-height:24px;
}

QLineEdit{
color:#1a1a1a;
font-size:16px;
line-height:24px;
padding-left:8px;
border:none;
border-bottom:1px solid #1a1a1a;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_33">
           <item>
            <widget class="QLabel" name="label_26">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>目的</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_s_goal">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_17">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>48</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CustomTableView" name="tableView">
        <property name="styleSheet">
         <string notr="true">QTableView::item:selected{
background:#12D94D;
}</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTableView</class>
   <extends>QTableView</extends>
   <header>widgets/customtableview/customtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
